lib/config/migrations/custom/regex-managers-migration.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { partial } from '../../../../test/util';
+import type { CustomManager } from '../../types';
+import { RegexManagersMigration } from './regex-managers-migration';
+
+describe('config/migrations/custom/regex-managers-migration', () => {
+  it('migrates', () => {
+    expect(RegexManagersMigration).toMigrate(
+      {
+        regexManagers: partial<CustomManager>([
+          {
+            fileMatch: ['js', '***$}{]]['],
+            matchStrings: ['^(?<depName>foo)(?<currentValue>bar)$'],
+            datasourceTemplate: 'maven',
+            versioningTemplate: 'gradle',
+          },
+          {
+            customType: 'jsonata',
```

Comment:
Admittedly besides the point of this review, but: this custom manager appears not to use JSONata. Why does it bear thar name?

Reply:
True, the other options are not in line with the jsonata manager which we will add. Will remove this for now and add it when `jsonata` is added.